futures/sequential-processing/web/main.dart

```diff
@@ -3,6 +3,7 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:html';
 
 void main() {
```

Comment:
Should we consider using `async` here?

https://dart.dev/guides/language/effective-dart/usage#prefer-asyncawait-over-using-raw-futures

Reply:
It is cleaner to do with `async` and `await`, but this directory of samples seems to be all about using Futures API.